applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -8,11 +8,13 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
     solver_type = solver_settings["solver_type"].GetString()
 
     if solver_type == "ale_fluid":
-        import navier_stokes_ale_fluid_solver
+        # This include NEEDS to be here or it will break everything.
```

Comment:
```suggestion
        # This include NEEDS to be here bcs of its dependencies
```

Reply:
Aww man, what do you have against generic apocalyptic messages?

Reply:
Nothing, but I am afraid that I cannot keep up with the masters of those => [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/ContactStructuralMechanicsApplication/custom_conditions/mortar_contact_condition.h#L586) and [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h#L630)